### ISSUE TITLE
Access Media Stream without Compliance Policy

### ISSUE THREAD

**Issue by aarushi-mittal** [microsoftgraph/microsoft-graph-comms-samples#182]
Hello Team,

I am trying to access media streams.

I wanted to inquire if its possible to access participant media streams without creating creating/assigning any compliance policy request.

**Comment by bijaykush** [comment]
@aarushi-mittal 
I don't think it is possible to get the media streams of a participant in a call without assigning a compliance recording policy.

But still, @ksikorsk  can provide you with the exact case.
